src/main/java/ChatReques6.java

```diff
@@ -0,0 +1,27 @@
+import lombok.*;
```

Comment:
Avoid using wildcard imports as they can lead to naming conflicts and make the code less readable. Specify the exact classes needed.